### PR TITLE
Deliveryfile revision

### PR DESCRIPTION
**Pull request by sylvinite**
### The purpose of the code changes are as follows:
-  Added multiqc html and multiqc json
- Added reasonable names for KS reports
- Added concat pangolin and consensus
- Hid raw Artic output and heavy files (vcf, bam)
- Moved VOC SNPs to easily editable txt

**How to prepare for test**:
- `cd MUTANT`
- `export PATH=$PATH:MUTANT_DIR`
- `source activate CONDA_ENV`
- `pip install -e .`

### How to test:
- `mutant analyse sarscov2 tests/testdata/fasta_files --profiles local,singularity --config_artic mutant/config/local/artic.json --config_mutant mutant/config/local/mutant.json --config_case tests/testdata/MIC3109_artic.json`

### Expected outcome:
- [x] Produced files contain expected values

### Review:
- [x] Code reviewed by @Mropat @talnor 

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by sylvinite** [comment]
Linked issue here https://github.com/Clinical-Genomics/hermes/pull/22

File: mutant/modules/sarscov2_delivery.py [hunk 1]
```diff
@@ -61,14 +63,28 @@ def rename_deliverables(self):
...
-                hit = glob.glob("{0}/*{1}".format(self.indir, thing))
-                if len(hit) == 1:
-                    hit = hit[0]
-                    os.symlink(hit, "{0}/{1}{2}".format(self.indir, self.ticket, thing))
+
+        ## Rename case files
+
+        # rename multiqc
+        hit = glob.glob("{}/multiqc/*_multiqc.html".format(self.indir))
+        if len(hit) == 1:
```
Comment (Mropat) [inline mutant/modules/sarscov2_delivery.py:71]: can just work with `if hit` cause [] == None

File: mutant/modules/sarscov2_report.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def create_sarscov2_resultfile(self):
         indir = self.indir
 
         summaryfile = os.path.join(
-            indir, "sars-cov-2_{}_results_{}.csv".format(ticket, today)
+            indir, "sars-cov-2_{}_results.csv".format(ticket, today)
```
Comment (Mropat) [inline mutant/modules/sarscov2_report.py:141]: remove todays date?

File: mutant/modules/sarscov2_report.py [hunk 3]
```diff
@@ -182,7 +182,7 @@ def create_sarscov2_variantfile(self):
 
         varRep = glob.glob(os.path.join(indir, "*variant_summary.csv"))[0]
         varout = os.path.join(
-            indir, "sars-cov-2_{}_variants_{}.csv".format(ticket, today)
+            indir, "sars-cov-2_{}_variants.csv".format(ticket, today)
```
Comment (Mropat) [inline mutant/modules/sarscov2_report.py:185]: same, if formatting is applied with only one value!

File: mutant/modules/sarscov2_report.py [hunk 4]
```diff
@@ -326,53 +326,124 @@ def create_deliveryfile(self):
...
-                "tag": "SARS-CoV-2-qc",
+                "step": "analysis",
+                "tag": "consensus",
+            }
+        )
+
+        # Multiqc report
+        deliv["files"].append(
+            {
+                "format": "json",
```
Comment (Mropat) [inline mutant/modules/sarscov2_report.py:381]:
```suggestion
                "format": "html",
```

File: mutant/modules/sarscov2_report.py [hunk 5]
```diff
@@ -326,53 +326,124 @@ def create_deliveryfile(self):
...
-                "tag": "SARS-CoV-2-var",
+                "tag": "pangolin-typing",
             }
         )
-        # QC report
+
+        # Consensus file
         deliv["files"].append(
             {
                 "format": "csv",
```
Comment (Mropat) [inline mutant/modules/sarscov2_report.py:369]:
```suggestion
                "format": "fasta",
```

File: mutant/modules/sarscov2_report.py (hunk 3 above)
Comment (Mropat) [inline mutant/modules/sarscov2_report.py:185]:
```suggestion
            indir, "sars-cov-2_{}_variants.csv".format(ticket)
```

File: mutant/modules/sarscov2_report.py [hunk 6]
```diff
@@ -326,53 +326,124 @@ def create_deliveryfile(self):
...
         deliv["files"].append(
             {
                 "format": "json",
                 "id": self.case,
-                "path": "{}/{}.json".format(self.indir, self.case),
+                "path": "{}/{}_multiqc.json".format(self.indir, self.ticket),
+                "path_index": "~",
+                "step": "report",
+                "tag": "multiqc-json",
+            }
```
Comment (Mropat) [inline mutant/modules/sarscov2_report.py:398]: dont forget to add multiqc-html to deliverables file!

File: mutant/modules/sarscov2_delivery.py (hunk 1 above)
Comment (sylvinite) [reply mutant/modules/sarscov2_delivery.py:71]: Nah cause we want to make sure there is just a single file.

**Pull request merged**